wiki/Tournaments/CWC/1/en.md

```diff
@@ -0,0 +1,265 @@
+- ![][flag_CA] [weezy47](/users/67211)
+- ![][flag_SG] [ZHSteven](/users/142413)
+- ![][flag_RU] [Louis Cyphre](/users/186243)
+- ![][flag_IT] [NeoRainier](/users/326049)
+
+## Links
+
+- [Discussion thread](/community/forums/topics/63296)
+- [Livestream](http://de.justin.tv/inoodle)
+  
+-----------------------
+
+## Participants
+
+| Team | Continent | Members |
+| :-- | :-- | :-- |
+| **#filipino** | Philippines | **blacksymbian**, Cielo, thOnz, jdle07 |
+| **CatchTheHungarians** | Europe | **Shauni8**, Kriszteeee, tuzgu, Norbi |
+| **CFC** | Asia/America | **KRZY**, Kuzino, ElectricShock, GuruK |
+| **Champion The Beat** | Taiwan, Korea | **Uan**, Bossandy, - Emp4y -, SS\_Kanzaki |
+| **Chile** | America | **eldnl**, Dark\_Diego, DaxMaster, MoodyRPG |
+| **China** | China | **hy1hy1hy**, HineX, Dusk, joynama |
+| **French fruit overdose club** | France | **Drafura**, Lendrix, Badis23, Tensude |
+| **German Hentai's** | Germany | **NoteKuroi**, DeathXHunter, Valetine, Nyan-Zapo |
+| **half manual player team** | Taiwan | **Flanet**, madaoB, allen22604358, H i M e J i |
+| **Hungary CTB** | Europe | **TaylorXIII**, Petya, Kuroi Mato, Felhoo |
+| **Indonesia** | Asia | **Lady Vamps**, -Newbie-, r3yn41d0, Ableh\_ZZZ |
+| **Indonesia2** | Indonesia | **Yumi14**, Varri Naufalb, AeroRange, Devil\_Cosmo |
+| **Korea** | Korea | **Spectator**, Speedy Cat, CTB user, CLSW |
+| **Lone wolfs** | North America | **sk8terpoke**, Gfbrd, Bolio, DeathxShinigami |
+| **Old School** | Europe | **CriticalDex**, Joseph, Emaal, Salwinjack |
+| **Red Fury** | Europe | **Deif**, mikhe, raimoncortese, AleZer0 |
+| **State(s) of the Art** | North/South America | **Senyo**, AroundTheManhole, Pieguy1372, k4mm3r |
+| **Team Nub** | Canada | **Sumaki**, Weezy, Kitsunemimi, jonnypui |
+| **The Hidden Ones** | Belarus, Czech Republic, Italy | **Katsek**, Jekasa, NeoRainier |
+| **Viciados em Crack** | South America | **Artificial Children**, Takamura, Kakarotto, Kazuo |
+
+## Groups
+
+| Group 1 | Group 2 | Group 3 | Group 4 |
+| :-- | :-- | :-- | :-- |
+| The Hidden Ones | State(s) of the Art | Team Nub | Chile |
+| Viciados em Crack | CFC | half manual player team | China |
+| Champion The Beat | Hungary CTB | Red Fury | French fruit overdose club |
+| Indonesia | #filipino | German Hentai's | CatchTheHungarians |
+| Lone wolf | Old School | Indonesia2 | Korea |
+
+---------------------------
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Team |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![][flag_CN] China |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_TW] half manual player team (Taiwan) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_CL] Chile |
+
+---------------------------
+
+## Mappools
+
+### Finals
+
+- NoMod
+  - [SUPER STAR -MITSURU- - THANK YOU FOR PLAYING \[CWC competition\]](/beatmaps/137035&m=2)
```

Comment:
Remove `&m=2` and unlink any broken beatmap links.

Reply:
&m=2 is needed to convert the leaderboards to ctb, better this since it show's ctb star rating rather than standard.

Reply:
does this beatmap exist? I get a 404

Reply:
@cl8n it doesn't exist, so I said to unlink any broken ones.
@Xinnoh it doesn't seem to work on new site, so

Use new links(e.g. `11620#fruits/44782`) now.